lib/routes/v1.js

```diff
@@ -105,6 +105,15 @@ api.declare({
     return;
   }
 
+  // Figure out which scopes the client needs
+  let requiredScopes = this.WorkerType.scopes(input);
+  // Now check for them if there are any
+  if (requiredScopes.length > 0) {
+    if (!req.satisfies(requiredScopes)) {
```

Comment:
I think this has to be `req.satisfies([requiredScopes])`..

The argument given to `satisfies` is a set of scope-sets in disjunctive form.
e.g. `req.satisfies([[a], [b,c]])` means "`a` or (`b` and `c`)"